### ISSUE TITLE
TODOs for v2 Release

### ISSUE THREAD

**Issue by felixguendling** [motis-project/motis#635]
- i18n web UI: https://github.com/motis-project/motis/issues/531
- docs: https://github.com/motis-project/motis/issues/562
- cleanup API:
  - https://github.com/motis-project/motis/issues/613
  - https://github.com/motis-project/motis/issues/612
  - https://github.com/motis-project/motis/issues/610
  - https://github.com/motis-project/motis/issues/630
- pass through nigiri parameters:
  - via stations
  - max travel time
  - max transfers
  - min transfer time
  - transfer time factor
- monitoring:
  - OTEL + Prometheus
- GBFS
  - #633: parallel network requests + caching
  - #632: support malformed GeoJSON
- osr: deal with no level <-> specified level transitions not marked with ramp/entry/etc.
  - problem: https://github.com/motis-project/motis/issues/626
  - wip solution: https://github.com/motis-project/osr/pull/22

**Comment by DomeQdev** [comment]
@felixguendling any update on trip modifications and added/new/duplicated?

**Comment by traines-source** [comment]
ADDED/NEW are supported, DUPLICATED/REPLACEMENT are work in progress, trip modifications is more long term. Do you have any example feeds that use these?

**Comment by DomeQdev** [comment]
Not really, but I'm planning to implement it for my [Warsaw GTFS-RT](https://cdn.zbiorkom.live/gtfs-rt/warsaw.pb) feed. If you would like, I can provide one random trip modification in this feed.

**Comment by traines-source** [comment]
Ah nice. Any reason you would do that with trip modifications instead of NEW/REPLACEMENT? As far as I recall the strengths of trip modifications were that one could modify many trips at once (i.e. being less complicated for producers, but more complicated for consumers). So I would lean towards limiting ourselves to REPLACEMENT/DUPLICATED in MOTIS for the time being.

**Comment by Jouca** [comment]
Hello @felixguendling,
Do you have any update about DUPLICATED/REPLACEMENT trip modifications?

I need it specifically for IDFM GTFS-RT for RATP lines.

**Comment by felixguendling** [comment]
Why would cancel + new not work in your case?

**Comment by Jouca** [comment]
It does, but based on the [GTFS-RT documentation](https://gtfs.org/fr/documentation/realtime/reference/#enum-schedulerelationship_1), it seems that it would be much better to do it like that.

Because currently, I need to cancel RATP lines due to how they works with realtime data and replace with new trips based on theoretical trips.

<img width="487" height="629" alt="Image" src="https://github.com/user-attachments/assets/fa2625bc-0e6d-424c-a7a8-804d5576e205" />

**Comment by Jouca** [comment]
There is also a difference between a CANCELED trip and a REPLACEMENT trip also, for mobility apps it's quite weird to make the difference with only CANCELED trips that they are just replaced and some that are really canceled.

**Comment by felixguendling** [comment]
We are still planning on implementing this but we didn't have time yet.

Do you have a feed to test with? A static GTFS and a single protobuf file is sufficient.

**Comment by Jouca** [comment]
You can use these feeds, I've applied the REPLACEMENT objects :

GTFS : https://clarifygdps.com/hexatransit/datasets/gtfs/fr-idf.zip
GTFS-RT : http://gtfsidfm.clarifygdps.com/gtfs-rt-trips-idfm-beta

**Comment by Jouca** [comment]
@felixguendling I created a new endpoint for the GTFS-RT with REPLACEMENT objects, use this one :

http://gtfsidfm.clarifygdps.com/gtfs-rt-trips-idfm-beta
